scripts/rbac_load_users.sh

```diff
@@ -29,7 +29,7 @@ CLIENT_ID="admin-cli"
 JSON_FILE="./data/rbac_users_data.json"
 
 TOKEN_RESPONSE=$(curl -s -X POST \
-  "${KEYCLOAK_ADMIN_ROUTE}/realms/${MASTER_REALM_NAME}/protocol/openid-connect/token" \
+  "${KEYCLOAK_ADMIN_ROUTE}realms/${MASTER_REALM_NAME}/protocol/openid-connect/token" \
```

Comment:
@vbelchio Was it intended to remove the slash? Is it broken right now? :) (add users part of the script appears to be running with no issues at least -- haven't tested it recently).

Reply:
@merlante thank you for seeing this, it was a typo. I am going to update it properly ;)

Reply:
Actually, it's working well. `KEYCLOAK_ADMIN_ROUTE` is coming with a slash at the end, so now we don't have two slashes (//) in the URL.